low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -118,46 +124,27 @@ def __init__(self):
-            self.encl_conf, COMMON_CONFIGS.get(self.encl_conf).get("secondary_controller_ip"), self.DEFAULT_MC_IP)
-        self.mc2_wsport = self.conf_reader._get_value_with_default(
-            self.encl_conf, COMMON_CONFIGS.get(self.encl_conf).get("secondary_controller_port"), '')
+        self.mc1 = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{CONTROLLER}>{PRIMARY}>{IP}", self.DEFAULT_MC_IP)
+        self.mc1_wsport = str(Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{CONTROLLER}>{PRIMARY}>{PORT}", ''))
+        self.mc2 = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{SECONDARY}>{IP}", self.DEFAULT_MC_IP)
+        self.mc2_wsport = str(Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{CONTROLLER}>{SECONDARY}>{PORT}", ''))
 
         self.active_ip = self.mc1
         self.active_wsport = self.mc1_wsport
 
-        self.user = self.conf_reader._get_value_with_default(
-            self.encl_conf, COMMON_CONFIGS.get(self.encl_conf).get("user"), self.DEFAULT_USER)
-        self.passwd = self.conf_reader._get_value_with_default(
-            self.encl_conf, COMMON_CONFIGS.get(self.encl_conf).get("password"), self.DEFAULT_PASSWD)
-
-        self.mc_interface = self.conf_reader._get_value_with_default(
-                                self.encl_conf, COMMON_CONFIGS.get(self.encl_conf).get("mgmt_interface"), "cliapi")
-
-        self.pollfreq = int(self.conf_reader._get_value_with_default(
-            self.CONF_REALSTORSENSORS, "polling_frequency", self.DEFAULT_POLL))
-
-        self.site_id = self.conf_reader._get_value_with_default(
-                                                self.SYSTEM_INFORMATION,
-                                                COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.SITE_ID),
-                                                '001')
-        self.rack_id = self.conf_reader._get_value_with_default(
-                                                self.SYSTEM_INFORMATION,
-                                                COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.RACK_ID),
-                                                '001')
-        self.node_id = self.conf_reader._get_value_with_default(
-                                                self.SYSTEM_INFORMATION,
-                                                COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID),
-                                                '001')
+        self.user = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{CONTROLLER}>{USER}", self.DEFAULT_USER)
+        self.passwd = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{CONTROLLER}>{SECRET}", self.DEFAULT_PASSWD)
+
+        self.mc_interface = Conf.get(SSPL_CONF, f"{STORAGE_ENCLOSURE}>{MGMT_INTERFACE}", "cliapi")
+
+        self.pollfreq = int(Conf.get(SSPL_CONF, f"{self.CONF_REALSTORSENSORS}>{POLLING_FREQUENCY}",
+                        self.DEFAULT_POLL))
+
+        self.site_id = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{self.SITE_ID}",'001')
```

Comment:
We can default default site/rack/node id, maintained in sspl_constants instead of 001 directly

Reply:
I have removed `001`  and ued new default IDs SN01, DC01 etc.